theme/themes/pastanaga/extras/blocks.less

```diff
@@ -470,18 +469,19 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
 }
 
 .ui.basic.button.block-add-button {
-  // .ui.basic.button.block-delete-button {
   position: absolute;
   z-index: 2;
-  // margin-top: -21px;
-  top: -2px;
-  padding: 0;
-  border: none;
+  bottom: -22px;
+  left: 50%;
+  padding: 0 !important;
+  border: none !important;
+  border-radius: 50% !important;
```

Comment:
@sneridagh I don't like needing to add all these `!important` rules. They are there to override some of the styles inherited from `.ui.basic.button`. Can I remove some or all of those classes since this is not formatted like a normal UI button?

Reply:
In semantic, since it's so especific from the default CSS itself, sometimes there's no other way... we gave up on that long time ago. The component is using the semantic button, you'll have to remove it and use a normal button, then define all the CSS from scratch to make that work.